swupd/swupd_test.go

```diff
@@ -169,6 +169,7 @@ func TestProcess(t *testing.T) {
 	}
 
 	// "progress" messages should be processed correctly
+	//nolint: lll // WONTFIX
```

Comment:
Does this work on a single line?
I thought we found that the nolint only applied to a variable, function, or the module?

Reply:
Yes it does... it is being applied to the variable jsonMsg